dali/base/dasds.cpp

```diff
@@ -3749,11 +3755,7 @@ int CSDSTransactionServer::run()
             {
                 // 1) Need to increment sds_requests here
                 // NB: it will never be decremented. This is total for life of this instance.
-                
-                // 2) Need to increment sds_active_requests here
-                // and ensure it's scoped, such that it is guaranteed
-                // to decrement when handling is complete.
-                // NB: most transactions are handled asynchronously, so decrement will need to happen on another thread
+                pSdsRequestsReceived->inc(1);
 
                 msgCount++;
```

Comment:
> This doesn't need to be atomic. It also replaces msgCount, so that code can be removed.

all references to msgCount should now be removed (2), as replaced by pSdsRequestsReceived

Reply:
Removed msgCount.